cfgmgr/vxlanmgr.cpp

```diff
@@ -526,5 +536,38 @@ bool VxlanMgr::deleteVxlan(const VxlanInfo & info)
     return true;
 }
 
-
+void VxlanMgr::clearAllVxlanDevices()
+{
+    std::string stdout;
+    const std::string cmd = std::string("") + IP_CMD + " addr";
```

Comment:
I think you may want to use `ip link `which will give you a smaller output as compared to `ip addr` and yet you get all interfaces

Reply:
Good idea~